### DISCUSSION TITLE
One time life payment subscription

### DISCUSSION THREAD

**Discussion by Amps-NA** [d4rken-org/sdmaid-se#804] [Q&A]
Hi Darken,

I was running version 1.52.0 beta SD Maid SE and I upgraded to the £5 one off life subscription on my firestick 4k mx which unlocked all the apps features.  However, when I Downloaded the latest update of this app version 1.62.0 Beta itis now asking me again to upgrade subscription wise so as to unlock the features which I already had in version 1.52.0 beta. So as stated in your app to get in touch if there were any issues. Well there is an issue that requires highlighting.                                                                                     Also, another issue I came across with the same app is:- if you download the from google playstore to your android phone and I did exact same subscription upgrade then downloaded app to afore mentioned firestick and found I had to pay to upgrade exact same subscription. So, so far I have paid twice for 2 £5 oneoff life subscriptions ??!!

**Comment by d4rken** [comment]
The purchases are tied to the Google account that is logged into Google Play.
Are you using different accounts? Buying the same one-time purchase again should not be possible. It will display "you already own this item" in this case.

**Comment by Amps-NA** [comment]
Hi Matthias,

In the end the current firestick version i have came from Github, your
download page through the Fdroid link. Could that be the issue?
When i originally downloaded the app from google play store to my phone i
upgraded it then used apps2fire to download the upgraded app to my
firestick. However, when i downloaded it to my firestick I had to pay to
upgrade it again so i did as firesticks dont run the google framework. I
think thats what the issue is, is firesticks dont run the google frame work
which is why i have to pay to keep upgrading maybe?
What do you think?

Amps Na.

**Comment by d4rken** [comment]
Ah okay, now I understand.

To keep it privacy friendly the upgrade/sponsor state of the non-Google version is not synced to any server or synchronized "via cloud". Only stored locally.

But this is not an issue as the GitHub version does not enforce the sponsorship.

If you go through it once and sponsor, just return to the app the next time, without (with a clear conscience as you already donated). It works by an "honor system".

You can sponsor/donate as much as you want (as often as you want) or as little as you want (down to "not at all").

**Comment by Amps-NA** [comment]
Hi Matthias,

Thank you for all your help and keep up the good work big HUGGSS


Amps

**Comment by mario0318** [comment]
I wound up having this same issue but decided to pay for another lifetime purchase with how much work you've put into the new version.
